manual/commands.md

```diff
@@ -0,0 +1,633 @@
+      nodejs : Create npm global directory      TAGS: [parsoid]
+      nodejs : Add npm_config_prefix bin directory to global $PATH.     TAGS: [parsoid]
+      nodejs : Ensure npm global packages are installed.        TAGS: [parsoid]
+      nodejs : Ensure npm global packages are at the latest release.    TAGS: [latest, parsoid]
+      parsoid : Get Parsoid repository  TAGS: [parsoid]
+      parsoid : Patch Parsoid so it allows image tags   TAGS: [parsoid]
+      parsoid : Ensure parsoid group exists     TAGS: [parsoid]
+      parsoid : Ensure parsoid user exists      TAGS: [parsoid]
+      parsoid : Ensure parsoid directory permissions    TAGS: [parsoid]
+      parsoid : Ensure Parsoid dependencies are latest  TAGS: [latest, parsoid, parsoid-deps]
+      parsoid-settings : Ensure localsettings.js present and up-to-date TAGS: [parsoid]
+      parsoid-settings : Ensure /etc/init.d/parsoid configured  TAGS: [parsoid]
+      parsoid-settings : Enable parsoid service TAGS: [parsoid]
+
+  play #13 (logging-servers): logging-servers   TAGS: [logging]
+    tasks:
+      set-vars : Set meza-core path variables   TAGS: [logging]
+      set-vars : If using gluster (app-servers > 1), override m_uploads_dir     TAGS: [logging]
+      set-vars : Set meza local public variables        TAGS: [logging]
+      set-vars : Get individual wikis dirs from localhost       TAGS: [logging]
+      set_fact  TAGS: [logging]
+      set-vars : Set meza local secret variables        TAGS: [logging]
+      meza-log : Check if server log database exists    TAGS: [logging]
+      meza-log : Set fact if server log database DOES exist     TAGS: [logging]
+      meza-log : Set fact if server log database DOES NOT exist TAGS: [logging]
+      meza-log : Import server log database structure   TAGS: [logging]
+      meza-log : Check if disk_space table exists       TAGS: [logging]
+      meza-log : Set fact if disk_space table DOES exist        TAGS: [logging]
+      meza-log : Set fact if disk_space table DOES NOT exist    TAGS: [logging]
+      meza-log : Create table disk_space if not exists  TAGS: [logging]
+
+  play #14 (all:!exclude-all:!load-balancers-unmanaged): all:!exclude-all:!load-balancers-unmanaged     TAGS: [cron]
+    tasks:
+      set-vars : Set meza-core path variables   TAGS: [cron]
+      set-vars : If using gluster (app-servers > 1), override m_uploads_dir     TAGS: [cron]
+      set-vars : Set meza local public variables        TAGS: [cron]
+      set-vars : Get individual wikis dirs from localhost       TAGS: [cron]
+      set_fact  TAGS: [cron]
+      set-vars : Set meza local secret variables        TAGS: [cron]
+      cron : Ensure cron is running (and enable it at boot)     TAGS: [cron]
+      cron : Ensure crontab file up-to-date     TAGS: [cron]
+      cron : Ensure runAllJobs.php in place     TAGS: [cron]
+      lineinfile        TAGS: [cron]
+      lineinfile        TAGS: [cron]
+      cron : Ensure crontab up-to-date from file        TAGS: [cron]
+```
```

Comment:
see above.  It is useful to know these things are available, but they shouldn't include *all* the output here.